### PR TITLE
Added Aync load of webfonts 

### PR DESCRIPTION
**Pull request by jhavmoeller**

### CODE REVIEW

**Comment by Dan9boi** [comment]
What about documentation?

File: gulp/config.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ module.exports = (function () {
 
     var projectPath = "./";
     var bowerPath = projectPath + "vendor/bower"; // remember to edit .bowerrc aswell (for CLI)
-    var distPath = projectPath + "dist";
+    var distPath = projectPath + "/../NCDP.Website/dist";
```
Comment (Dan9boi) [inline gulp/config.js]: This shouln't be committed...

File: less/base/base.fonts.less [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+	src: local('☺'),
+		url('/dist/fonts/SkipLegDay.woff') format('woff'),
+		url('/dist/fonts/SkipLegDay.ttf') format('truetype'),
+		url('/dist/fonts/SkipLegDay.svg') format('svg');
+	font-weight: normal;
+	font-style: normal;
+}
+
+
+@font-face {
```
Comment (Dan9boi) [inline less/base/base.fonts.less]: Maybe we could do with just one font-face example?

File: less/base/base.typography.less [hunk 3]
```diff
@@ -47,3 +47,13 @@ ul {margin-bottom: 20px;}
 
 p~h2,
 p~h3 {margin-top: 36px;}
+
+
+html {
```
Comment (Dan9boi) [inline less/base/base.typography.less]: Comment for this part?

File: less/settings/settings.variables.less [hunk 4]
```diff
@@ -32,11 +32,11 @@
 
 /* Typography
 ********************/
-@base-font-family:              "Lato", "Helvetica", "Arial", "sans-serif";
+@base-font-family:              'Roboto', sans-serif;
```
Comment (Dan9boi) [inline less/settings/settings.variables.less]: I guess that we could still need the old fallback fonts.

File: gulp/config.js (hunk 1 above)
Comment (jhavmoeller) [inline gulp/config.js]: /fixed

File: less/base/base.fonts.less (hunk 2 above)
Comment (jhavmoeller) [inline less/base/base.fonts.less]: /fixed

File: less/base/base.typography.less (hunk 3 above)
Comment (jhavmoeller) [inline less/base/base.typography.less]: /fixed

File: less/settings/settings.variables.less (hunk 4 above)
Comment (jhavmoeller) [inline less/settings/settings.variables.less]: /fixed

**Comment by Dan9boi** [comment]
Great job! :100:

**Pull request merged**